### ISSUE TITLE
Finish partially implemented EBF bosses

### ISSUE THREAD

**Issue by chrisinajar** [OpenAngelArena/oaa#1340]
Related to #1281

8 bosses have been imported to OAA, most don't work. Lets get them working!

#### Factory
The maximum techie summon number is global instead of per boss, also still summons when on one is around which makes for a funny prank.

**Comment by Chronophylos** [comment]
The unit limit is set in kv
```
        "UnitCount"      "1"
        "UnitLimit"      "4"
```

**Comment by chrisinajar** [comment]
indeed, and it is enforced globally. this means you can aggro 1 factory, get it to spawn 4 techies, then go and solo the other factory because he'll never spawn any because there's already 4 at the other one

**Comment by Chronophylos** [comment]
tbh we should rewrite all of the ebf abilities as `ability_lua` not `ability_datadriven`

**Comment by zelding** [comment]
is this still relevant?

**Comment by l34Um1** [comment]
Yeah we still want those bosses

**Comment by carlosrpg** [comment]
is this still relevant?

**Comment by Haganeko** [comment]
I think so? No one has worked on it for ages, though.

**Comment by carlosrpg** [comment]
that's my point, do we want to blind port it or do we want to create the new designs? or Both?
@l34Um1 ?

**Comment by Haganeko** [comment]
Even if we want new bosses, these would at least do to increase our roster of bosses while the others are designed and implemented.
Same as we did with the Simple Bosses.

**Comment by l34Um1** [comment]
I don't see why we wouldn't want both. These bosses would definitely add something to the game. But if I had to choose I rather get the new ones.
